maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -454,6 +454,10 @@ class IOSDriver(
         LOGGER.warn("Airplane mode is not available on iOS simulators")
     }
 
+    override fun installApk(apk: File) {
+        LOGGER.warn("Install app is not available on iOS simulators")
+    }
+
```

Comment:
We want Maestro to be cross-platform wherever possible, so this won't fly. Let's add support for installing on iOS simulator as well.

We can use `xcrun` for that:

```
xcrun simctl install <device> <path-to-app>
```

for example:

```
xcrun simctl install booted build/ios/iphonesimulator/Runner.app
```

Reply:
Hello @bartekpacia thanks for your review! I also added for iOS.